.gitlab-ci.yml

```diff
@@ -22,10 +22,12 @@ cache:
 
 variables:
   GIT_SUBMODULE_STRATEGY: recursive
+  CODECOV_TOKEN: f3729c36-5ce7-48a3-b940-92f1cefd85a5
```

Comment:
I'm guessing this lets anybody with this token upload code coverage statistics for our repo? It's not a huge deal (seems like pretty petty trolling), but we should probably generate a new token and put it in GitLab's secret variables when this is ready to merge.